config/routes.js

```diff
@@ -89,14 +89,14 @@ module.exports.routes = {
   '/game/chat': 'game/chat',
   '/game/gameData': 'game/game-data',
   // DEVELOPMENT ONLY
-  '/game/stackDeck': 'game/stack-deck',
-  '/game/loadFixture': 'game/load-fixture',
-  '/game/clear': 'game/clear-game',
 
   // Testing helpers
   '/test/wipeDatabase': 'TestController.wipeDatabase',
   '/test/badSession': 'TestController.setBadSession',
   '/test/loadSeasonFixture': 'TestController.loadSeasonFixture',
   '/test/loadMatchFixtures': 'TestController.loadMatchFixtures',
   '/test/loadFinishedGameFixtures': 'TestController.loadFinishedGameFixtures',
+  '/test/stackDeck': 'TestController.stackDeck',
+  '/test/loadGameFixture': 'TestController.loadGameFixture',
+  '/test/clearGame': 'TestController.clear',
```

Comment:
This does seem like a good change (making the urls more consistent with the other test-specific ones), but it also means that we need to change all references to these urls. Look for all the consumers of
```
/game/stackDeck
/game/loadFixture
/game/clear
```

Looking at them now, both `stackDeck` and `clearGame` are actually unused in the entire app. So we can delete those two altogether, (both the files themselves, + the route and policy mappings).

Whereas the `'/game/LoadFixture'` route is currently in use in `tests/e2e/support/commands.js` in the `loadGameFixture()` command. We need to make sure that this function is updated so that it calls the updated endpoint url i.e. `'/test/loadGameFixture'`
```suggestion
  '/test/loadGameFixture': 'TestController.loadGameFixture',
```